pkg/auth/handler/oauth/authorize.go

```diff
@@ -52,14 +50,10 @@ func (h *AuthorizeHandler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
 	}
 
 	var result httputil.Result
-	err = h.Database.WithTx(ctx, func(ctx context.Context) error {
-		result = h.AuthzHandler.HandleRequestWithTx(ctx, req, params)
-		if result.IsInternalError() {
-			return errAuthzInternalError
-		}
-		return nil
-	})
-
+	result = h.AuthzHandler.HandleRequestWithTx(ctx, req, params)
```

Comment:
Because there is no Tx here, maybe we should rename it to `HandleRequest`.

Reply:
Addressed.